### ISSUE TITLE
Private query fns ignore the result value

### ISSUE THREAD

**Issue by codahale** [layerware/hugsql#32]
I have a Postgres query which looks like this:

```sql
-- :name add-thing :<! :1
update things
set count = count + 1
where id = :id
returning count;
```

It works fine, but when I try to make the query fn private (i.e., using `:name-`), it ignores the `:1` and treats the result value as `:raw`.

**Comment by csummers** [comment]
Nice catch--definitely a bug.  I've fixed this in master and it will roll in the next release.  Thanks for the report!

**Comment by csummers** [comment]
This is released in 0.4.6.  Please confirm that it fixes the issue for you.  Thanks!

**Comment by codahale** [comment]
Works great, thanks!
